### PR TITLE
Implement Team Set Serializer

### PR DESCRIPTION
**Pull request by Hedgemon4**
Description:
- implemented team set serializer

Issue:
#156

### CODE REVIEW

File: tests/test_api/test_models/test_team_set/test_serializer.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        json_project_requirement_team_2 = (
+            '{"attribute": 1, "operator": "less than", "value": 1}'
+        )
+        json_team_1 = (
+            '{"_id": 5, "name": "Team Numero 5", "project_id": null, "requirements": [], "students": [%s, %s]}'
+            % (json_student_1, json_student_2)
+        )
+        json_team_2 = (
+            '{"_id": 2, "name": "Goosed", "project_id": 11, "requirements": [%s], "students": [%s]}'
+            % (json_project_requirement_team_2, json_student_3)
```
Comment (Opeyem1a) [inline tests/test_api/test_models/test_team_set/test_serializer.py:47]: why not use f-strings here?

**Review by Opeyem1a — CHANGES_REQUESTED** [review]
looks good, one comment though

File: tests/test_api/test_models/test_team_set/test_serializer.py (hunk 1 above)
Comment (Hedgemon4) [reply tests/test_api/test_models/test_team_set/test_serializer.py:47]: Is the preference for f string?

Comment (Hedgemon4) [reply tests/test_api/test_models/test_team_set/test_serializer.py:47]: Does it not accomplish the same thing?

**Review by Opeyem1a — APPROVED** [review]

**Pull request merged**